C/plugins/storage/postgres/connection.cpp

```diff
@@ -3312,3 +3312,240 @@ const char *p;
 		return true;
 	return false;
 }
+
+/**
+ * Create scema of tables
+ *
+ * @param payload   The  payload containing information about schema of 
+ *                  tables to create
+ * @return true if the tables can be crated successfully
+ */
+int Connection::create_schema(std::string payload)
+{
+
+	Document document;
+	std::string schema;
+	int version;
+	const char *logSection="CreatingSchema";
+	unsigned long rowsAffectedLastCommand = 0;
+
+	std::string queryToFindVerFromDB = "select * from fledge.version;" ;
```

Comment:
This is the fledge database schema ID and not the version of the extended schema. We can not assume the two are the same as the fledge database schema version only changes when the fledge schema is updated. The extended schemas will have their own versions independent of the fledge database schema.

I would suggest having a new table in the fledge data a that track the extended schemas that are created by the services that require them. This will require a one time update to the fledge schema to add this new table, but after that when a new version of an extended schema is required the fledge schema version will remain unchanged.